articles/resources/relnotes.md

```diff
@@ -17,6 +17,23 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.12.20092342
+
+*Release date: September 29th, 2020*
+
+This release contains the following:
+
+- Performance improvements for simulator: better application of gate fusion decisions; improved parallelization on Linux system; added intelligent scheduling of gate execution; bug fixes; overall speed improvement of ~5x (even faster on larger problems).
+- IntelliSense features are now supported for Q# files in Visual Studio and Visual Studio Code even without a project file.
+- Various Q#/Python interoperability improvements and bug fixes, including better support for NumPy data types.
+- Improvements to the Microsoft.Quantum.Arrays namespace (microsoft/QuantumLibraries#313)
+- Added a new language repo (QIR blog)[TODO]
+- General bug fixes.
+
+Since the last release, the default branch in each of our open source repositories has been renamed to `main`.
+
+See the full list of closed PRs for [libraries](https://github.com/Microsoft/QuantumLibraries/pulls?q=is%3Apr+is%3Aclosed), [compiler](https://github.com/microsoft/qsharp-compiler/pulls?q=is%3Apr+is%3Aclosed), [runtime](https://github.com/microsoft/qsharp-runtime/pulls?q=is%3Apr+is%3Aclosed), [samples](https://github.com/Microsoft/Quantum/pulls?q=is%3Apr+is%3Aclosed), [IQ#](https://github.com/microsoft/iqsharp/pulls?q=is%3Apr+is%3Aclosed) and [Katas](https://github.com/microsoft/QuantumKatas/pulls?q=is%3Apr+is%3Aclosed).
```

Comment:
```suggestion
See the full list of closed PRs for [libraries](https://github.com/Microsoft/QuantumLibraries/pulls?q=is%3Apr+closed%3A2020-09-02..2020-09-25+), [compiler](https://github.com/microsoft/qsharp-compiler/pulls?q=is%3Apr+closed%3A2020-09-02..2020-09-25+), [runtime](https://github.com/microsoft/qsharp-runtime/pulls?q=is%3Apr+closed%3A2020-09-02..2020-09-25+), [samples](https://github.com/Microsoft/Quantum/pulls?q=is%3Apr+closed%3A2020-09-02..2020-09-25+), [IQ#](https://github.com/microsoft/iqsharp/pulls?q=is%3Apr+closed%3A2020-09-02..2020-09-25+) and [Katas](https://github.com/microsoft/QuantumKatas/pulls?q=is%3Apr+closed%3A2020-09-02..2020-09-25+).
```

Reply:
Added the filter `is:pr closed:2020-09-02..2020-09-25` (code complete for August was on Wed 09-02)

Reply:
Actually the August release was on 8/25 ([see here](https://docs.microsoft.com/en-us/quantum/resources/relnotes#version-01220082513)), so perhaps we can use `2020-08-24..2020-09-24` as the date range?

Reply:
But I like this idea a lot, it will be really helpful if someone actually wants to see what was in the release!

Reply:
@rmshaffer thanks, updated the dates :)